pontoon/administration/static/css/admin_project.css

```diff
@@ -469,3 +469,7 @@ textarea.strings-source {
   width: 156px;
   margin-top: 0;
 }
+
+.tag.inline .tag-resources {
+    padding: 5px;
+}
```

Comment:
I don't think we need that, but I'll followup with CSS fixes anyways, so you can leave it.

Reply:
ill remove and reduce noize on PR - and you can dress it with a clean slate